### PR TITLE
Revive all command

### LINKED ISSUE #26070: group revive
**Description:**

how about create one command group revive to resurrect all players on a group?

**Expected behaviour:**

all group revives

**Branch(es):**

3.3.5

**TC rev. hash/commit:** 

bf0975f658d5

**Operating system:** 

N/R

### PR DESCRIPTION
**Pull request by J-Cieplinski**
**Changes proposed:**

-  Adds group revive command

**Target branch(es):** 3.3.5/master

- [x] 3.3.5
- [ ] master

**Issues addressed:**

Closes #26070 

**Tests performed:**

- [x] Does it build
- [x] Tested in world server console on both raid groups and 5 man groups

### CODE REVIEW

File: src/server/scripts/Commands/cs_misc.cpp [hunk 1]
```diff
@@ -110,6 +110,7 @@ class misc_commandscript : public CommandScript
             { "repairitems",      rbac::RBAC_PERM_COMMAND_REPAIRITEMS,       true, &HandleRepairitemsCommand,      "" },
             { "respawn",          rbac::RBAC_PERM_COMMAND_RESPAWN,          false, &HandleRespawnCommand,          "" },
             { "revive",           rbac::RBAC_PERM_COMMAND_REVIVE,            true, &HandleReviveCommand,           "" },
+            { "reviveall",        rbac::RBAC_PERM_COMMAND_REVIVEALL,         true, &HandleReviveAllCommand,        "" },
             { "saveall",          rbac::RBAC_PERM_COMMAND_SAVEALL,           true, &HandleSaveAllCommand,          "" },
```
Comment (Aokromes) [inline src/server/scripts/Commands/cs_misc.cpp:114]: imho better a subcommand to group.

Comment (J-Cieplinski) [reply src/server/scripts/Commands/cs_misc.cpp:114]: Could you elaborate?

Do you mean as not to create a new `RBAC_PERM_COMMAND_REVIVEALL` permission but just use existing `RBAC_PERM_COMMAND_REVIVE`?

Comment (Aokromes) [reply src/server/scripts/Commands/cs_misc.cpp:114]: i mean instead reviveall put it into cs_group and name it group revive.

Comment (J-Cieplinski) [reply src/server/scripts/Commands/cs_misc.cpp]: Oh sorry, I'm new to codebase. I see what you mean. Pushed

File: src/server/scripts/Commands/cs_group.cpp [hunk 2]
```diff
@@ -66,6 +67,29 @@ class group_commandscript : public CommandScript
         return commandTable;
     }
 
+
```
Comment (Aokromes) [inline src/server/scripts/Commands/cs_group.cpp:70]: remove this row

File: src/server/scripts/Commands/cs_group.cpp [hunk 3]
```diff
@@ -66,6 +67,28 @@ class group_commandscript : public CommandScript
...
+        if (!handler->extractPlayerTarget((char*)args, &playerTarget, &playerTargetGuid))
+            return false;
+
+        Group* groupTarget = playerTarget->GetGroup();
+        if (!groupTarget)
+            return false;
+
+        for (GroupReference* it = groupTarget->GetFirstMember(); it != nullptr; it = it->next())
+        {
+            Player* target = it->GetSource();
```
Comment (mdX7) [inline src/server/scripts/Commands/cs_group.cpp:83]: nullcheck - imagine someone goes offline

**Comment by matanshukry** [comment]
Voting for cherry-pick/another PR to master

**Comment by jackpoz** [comment]
Thanks for the PR :)

**Comment by jackpoz** [comment]
this PR caused a crash, see https://github.com/TrinityCore/TrinityCore/issues/26575 (the issue is from ".group repair" but the affected code is the same)

**Pull request merged**